portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,35 @@
-#content {
+
+.container{
+    height: 100vh;
+    width: 100%;
+    background-image: url(Images/background.jpg);
+    background-position: center;
+    background-size: cover;
+    box-sizing: border-box;
+    position: relative;
+}
+
+#intro {
   margin-left: auto;
   margin-right: auto;
   width: 650px;
 }
 
+
+
 #greeting-container {
   margin-top: 20px;
 }
+
+#photosofme {
```

Comment:
Awesome job naming! Follows guidelines here perfectly: https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Naming. Nitpick: words should be separated by "-", see https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters.